test/mail_helper.js

```diff
@@ -78,6 +78,17 @@ module.exports = (printLogs) => {
             } else {
               users[name] = [mail]
             }
+
+            if (mail.headers.cc) {
+              // Support for CC headers
+              var ccName = emailName(mail.headers.cc)
+
+              if (users[ccName]) {
+                users[ccName].push(mail)
+              } else {
+                users[ccName] = [mail]
+              }
+            }
```

Comment:
It's based on the same style from above (for .to field). Both can be simplified 👍